css/screen.scss

```diff
@@ -140,6 +140,13 @@ body.info {
   color: #888;
   font-size: 14px;
   font-weight: 100;
+  a {
+    width: 30px;
```

Comment:
Do you need this?

Reply:
That is how I don't make the link span the whole page - make it the same width as the svg.

Reply:
Are you sure you need it?

Reply:
Yes